File: 240115/Top K 숫자 구하기/kth-number.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main() {
    int a,b;
    cin >> a >> b;
    int arr[a];
    for(int i = 0; i < a; i++){
        cin >> arr[i];
    }
    sort(arr, arr+a);
    cout << arr[b-1];
    return 0;
}

File: 240602/Top K 숫자 구하기/kth-number.js
let fs = require("fs");

let input = fs.readFileSync(0).toString().split("\n");
let [n , k] = input[0].split(" ").map(Number);
let arr = input[1].split(" ").map(Number).sort((a , b) => a - b);
console.log(arr[k - 1]);